tests/general/base.py

```diff
@@ -0,0 +1,126 @@
+from test_plus import APITestCase
+from django.core.cache import cache
+
+from tests.profile.factories import (
+    ProfileFactory, ProfileHighlightFactory,
+    ProfileKeyMetricsFactory, IndicatorCategoryFactory,
+    IndicatorSubcategoryFactory
+)
+from tests.points.factories import (
+    CategoryFactory, LocationFactory
+)
+from tests.datasets.factories import (
+    GeographyHierarchyFactory,
+    VersionFactory, DatasetFactory,IndicatorFactory
+)
+from tests.boundaries.factories import GeographyBoundaryFactory
+from wazimap_ng.datasets.models import Geography
+from wazimap_ng.datasets.tasks.indicator_data_extraction import (
+    indicator_data_extraction
+)
+from wazimap_ng.datasets.tasks.process_uploaded_file import process_csv
+from tests.datasets.tasks.test_process_uploaded_file import create_datasetfile
+
+
+class ConsolidatedProfileViewBase(APITestCase):
+
+    def setUp(self):
+        """
+        Create Data for default_version
+        """
+        self.default_version = VersionFactory(name="default_version")
```

Comment:
```suggestion
        self.default_version = VersionFactory(name="v1")
```

Reply:
I think it's worth not conflating GeographyHierarchy.config.default_version with a version named "default_version".

When I see a version named v1, I assume that's the first (and only) version.

I think if tests that use a second Version create a version named v2, it should be quite clear that that's the second version in the system.